### PR TITLE
[CircleGraph] Relocate message handlers

### PR DESCRIPTION
**Pull request by seanshpark**
This will relocate message handler related codes to CircleGraphCtrl.

ONE-vscode-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for #641, from draft #647

File: src/CircleGraph/CircleGraph.ts [hunk 1]
```diff
@@ -76,24 +60,19 @@ export class CircleGraphPanel extends CircleGraphCtrl {
 
     // Otherwise, create a new panel.
     const panel = vscode.window.createWebviewPanel(
-        CircleGraphPanel.viewType,
-        'CircleGraphPanel',
-        column || vscode.ViewColumn.One,
-        getWebviewOptions(extensionUri),
-    );
+        CircleGraphPanel.viewType, 'CircleGraphPanel', column || vscode.ViewColumn.One,
+        {retainContextWhenHidden: true});
```
Comment (hyunsik-yoon) [inline src/CircleGraph/CircleGraph.ts:64]: Shouldn't this be `CircleGraphCtrl.getWebviewOptions()` ?

Comment (seanshpark) [reply src/CircleGraph/CircleGraph.ts:64]: I think I wanted to do something with `getWebviewOptions` but I guess I'll do that in another PR.

**Review by hyunsik-yoon — APPROVED** [review]

**Comment by seanshpark** [comment]
@hyunsik-yoon , rebased. PTAL again;

**Review by hyunsik-yoon — APPROVED** [review]

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**